sui_core/src/consensus_adapter.rs

```diff
@@ -195,9 +195,11 @@ impl ConsensusListener {
                     match message {
                         // Keep track of this certificates so we can notify the user later.
                         ConsensusListenerMessage::New(transaction, replier) => {
+                            let digest = Self::hash(&transaction);
                             if self.pending.len() < self.max_pending_transactions {
-                                let digest = Self::hash(&transaction);
                                 self.pending.entry(digest).or_insert_with(Vec::new).push(replier);
+                            } else if replier.send(Err(SuiError::ListenerCapacityExceeded)).is_err() {
+                                debug!("No replier to listen to consensus output {digest}");
```

Comment:
Is this a `debug` or a `warn`? 

Do we have an issue / comment to track the recovery logic, and which side is involved (Narwhal, Sui ...).

Reply:
I try to never allow bad clients to make us log warns (only bad does)

Reply:
the recovery logic will have to be the retries of the client

Reply:
A way to do this orthogonally to the log level is to set up a span, with context on what's going on. That would allow us to filter what's going on:
https://github.com/MystenLabs/sui/blob/1a25e68fbc849684344680a4da134d6a62b136c3/sui_core/src/authority_server.rs#L169-L174